go/vt/vtgate/planbuilder/route.go

```diff
@@ -533,6 +536,11 @@ func (rb *route) Wireup(bldr builder, jt *jointab) error {
 	return nil
 }
 
+func systemTable(qualifier string) bool {
+	lowered := strings.ToLower(qualifier)
+	return lowered == "information_schema" || lowered == "performance_schema" || lowered == "sys"
```

Comment:
I could be wrong, but I think strings.EqualFold may be more performant here. It avoids creating a new string like ToLower, and can abort early in the string if it's not a match.

Reply:
oooh! i like it